dynatrace-activegate-operator-google-marketplace/charts/dynatrace-activegate-operator/templates/rolebinding-activegate.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
This and a few other files are missing the license header.

Reply:
Added for all files